Python/Product/Cookiecutter/Model/CookiecutterClient.cs

```diff
@@ -141,6 +147,26 @@ class CookiecutterClient : ICookiecutterClient {
             MoveToDesiredFolder(outputFolderPath, tempFolder);
         }
 
+        public Task<string> GetDefaultOutputFolderAsync(string shortName) {
```

Comment:
This doesn't quite fit in with the rest of CookiecutterClient's responsibilities, which is to load/execute templates (interfacing with the python package).  As long as we don't start dealing with VS projects and the like, it can stay here, and move it out when necessary.